includes/pages/admin_shifts.php

```diff
@@ -339,9 +339,9 @@ function admin_shifts()
                 $shifts_table_entry = [
                     'timeslot'      =>
                         icon('clock-history') . ' '
-                        . $start->format(__('general.datetime'))
+                        . dateWithEventDay($start->format('Y-m-d')) . ' ' . $start->format(__('H:i'))
                         . ' - '
-                        . '<span title="' . $end->format(__('general.date')) . '">'
+                        . '<span title="' . dateWithEventDay($end->format(__('general.date'))) . '">'
```

Comment:
That can't be translated here or it breaks when viewing a shift on the german site:

```
InvalidFormatException
Unexpected data found. Unexpected data found. Trailing data
```